### PR TITLE
Add missing backtick in defaults table

### PR DESCRIPTION
**Pull request by tedmiston**
**What this PR does / why we need it**: Fixes a current typo in markdown formatting.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by tedmiston** [comment]
/retest

The failed test on Jenkins seems unrelated to the doc change of this PR.  Let's try re-running it.

**Comment by tedmiston** [comment]
@timstoop @kocolosk Any idea about the CrashLoopBackOff causing pull-charts-e2e to fail?

https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/charts/4528/pull-charts-e2e/7374/

I reviewed the logs and haven't found anything obvious yet.

**Comment by tedmiston** [comment]
Rebased off master to see if it fixes the random failing test.

/retest

**Comment by tedmiston** [comment]
@timstoop @unguiculus I think this test failure may be a flake caused by the same open issue as #3506.

**Comment by unguiculus** [comment]
@tedmiston It's not related to #3506.

**Comment by tedmiston** [comment]
Thank you for checking.  Can you give any advice about what to do move this forward?  This PR changes just docs (no code change), so a breaking test seems weird.

**Comment by unguiculus** [comment]
@kocolosk Could you have a look, please?

**Comment by timstoop** [comment]
From the error it looks like the image tries to change the ownership of files made available via ConfigMap, which is no longer allowed:
```
I0412 18:15:40.150] chown: changing ownership of '/opt/couchdb/etc/default.d/..data': Read-only file system
I0412 18:15:40.150] chown: changing ownership of '/opt/couchdb/etc/default.d/chart.ini': Read-only file system
I0412 18:15:40.150] chown: changing ownership of '/opt/couchdb/etc/default.d/..2018_04_12_17_57_07.264705909/chart.ini': Read-only file system
I0412 18:15:40.150] chown: changing ownership of '/opt/couchdb/etc/default.d/..2018_04_12_17_57_07.264705909': Read-only file system
I0412 18:15:40.150] chown: changing ownership of '/opt/couchdb/etc/default.d': Read-only file system
```

**Comment by kocolosk** [comment]
Thanks @timstoop that makes sense. Will see if I can pull together a fix tomorrow.

**Comment by kocolosk** [comment]
@tedmiston the underlying issue causing the CrashLoopBackOff has been fixed in #5044 - can you rebase this?

File: incubator/couchdb/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name: couchdb
-version: 0.1.0
+version: 0.1.1
+appVersion: 0.1.1
```
Comment (kocolosk) [inline incubator/couchdb/Chart.yaml]: `appVersion` should be 2.1.1

**Comment by kocolosk** [comment]
FYI this is also addressed by 2ddd5ba in #5640

**Comment by tedmiston** [comment]
@kocolosk Thank you.  Here's a fresh rebase which is now green on the CI for the test that was previously failing.

@unguiculus @timstoop Are we good to merge now?

Thanks again all for your assistance as I get used to this process.

**Comment by kocolosk** [comment]
Thanks @tedmiston, all set from my end.

**Pull request merged**